CONTRIBUTING.md

```diff
@@ -47,24 +47,28 @@ https://github.com/jonschlinkert/markdown-toc
 ## Code development
```

Comment:
One more comment:
I think Documentation section needs a bit of a refresh.

We dont need to install `mkdocs_requirements.txt` anymore, and a description how to test the docs would help

Reply:
I think that one is going to be implemented in #81